astro.config.ts

```diff
@@ -22,6 +22,12 @@ export default defineConfig({
 				replacesTitle: true,
 			},
 			customCss: ['./src/styles/custom.css'],
+			head: [
+				{
```

Comment:
Can you add a link to the related docs? Just to make it easier to update if needed